graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -88,19 +89,15 @@ def get_superclasses_from_class_definition(class_definition):
     return []
 
 
+@six.add_metaclass(ABCMeta)
 class SchemaElement(object):
```

Comment:
We probably want `@six.python_2_unicode_compatible` here as well, check out the docs for it.